src/util/exportHelpers.ts

```diff
@@ -0,0 +1,74 @@
+import { Attendee } from "@/components/RegistrationTable/columns";
+import { Table } from "@tanstack/react-table";
+import pdfMake from 'pdfmake/build/pdfmake';
+import pdfFonts from 'pdfmake/build/vfs_fonts';
+import { PageOrientation } from "pdfmake/interfaces";
+
+pdfMake.vfs = pdfFonts.pdfMake.vfs;
+
+export function exportToCSV(data: Attendee[], fileName = 'data') {
+    if (!data.length) {
+        console.error("No data to export");
+        return;
+    }
```

Comment:
Good conditional to prevent crashes - I'm wondering if we should go a step further and prevent the export options from appearing if there's no data in the table?

Reply:
Good idea I conditionally disabled the export button if the table has no rows